hpx/runtime/actions/base_action.hpp

```diff
@@ -112,6 +112,7 @@ namespace hpx { namespace actions
             was_object_migrated(hpx::naming::gid_type const&,
                 naming::address_type) = 0;
 
+#if defined(HPX_HAVE_NETWORKING)
         /// Return a pointer to the filter to be used while serializing an
```

Comment:
Hmmm, I was sure that we can get rid of the entire `base_action` and `transfer_action` business.

Reply:
Didn't we?

Reply:
no, it is still completely there, it is not `#ifdef`-ed out completely

Reply:
That's removed now.